internal/proto/controller/api/resources/users/v1/user.proto

```diff
@@ -11,61 +11,61 @@ import "controller/custom_options/v1/options.proto";
 
 message Account {
   // Output only. The ID of the Account.
-  string id = 10;
+  string id = 10; // @gotags: `class:"public"`
 
   // Output only. The Scope containing the Account.
-  string scope_id = 20 [json_name = "scope_id"];
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
 }
 
 // User contains all fields related to a User resource
 message User {
   // Output only. The ID of the User.
-  string id = 10;
+  string id = 10; // @gotags: `class:"public"`
 
   // The ID of the Scope this resource is in.
-  string scope_id = 20 [json_name = "scope_id"];
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
 
   // Output only. Scope information for this resource.
   resources.scopes.v1.ScopeInfo scope = 30;
 
   // Optional name for identification purposes.
-  google.protobuf.StringValue name = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "name" that: "name" }];
+  google.protobuf.StringValue name = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "name" that: "name" }]; // @gotags: `class:"public"`
 
   // Optional user-set description for identification purposes.
-  google.protobuf.StringValue description = 50 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "description" that: "description" }];
+  google.protobuf.StringValue description = 50 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "description" that: "description" }]; // @gotags: `class:"public"`
```

Comment:
This should maybe be sensitive. Less likely than name to be PII but still possible. Happy either way.